x-pack/plugins/security_solution/server/lib/detection_engine/rules/bulk_actions/rule_params_modifier.ts

```diff
@@ -36,7 +35,7 @@ const applyBulkActionEditToRuleParams = (
         "Index patterns can't be added. Machine learning rule doesn't have index patterns property"
       );
 
-      if (!isMachineLearningParams(ruleParams) && action.overwriteDataViews) {
```

Comment:
why did we remove the isMachineLearningParams check?

Reply:
there is a check for rule type, a line above

https://github.com/elastic/kibana/blob/2d72d249085471514b57111f2b16340d7bc2ef28/x-pack/plugins/security_solution/server/lib/detection_engine/rules/bulk_actions/rule_params_modifier.ts#L33-L36


if it's ML, it will throw an error. So, another ML check is redundant there